modules/crowdfundings/graphql/resolvers/_queries/paymentsCSV.js

```diff
@@ -154,18 +156,18 @@ module.exports = async (_, args, {pgdb, user}) => {
       pledgeId: result.pledgeId.substring(0, 13),
       userId: result.userId.substring(0, 13),
       email: result.email,
-      firstName: result.firstName,
-      lastName: result.lastName,
+      firstName: result.firstName.replace(/,/gi, ''),
+      lastName: result.lastName.replace(/,/gi, ''),
```

Comment:
this should not be necessary, `dsvFormat` should handle this cases. Also, this export is `;` separated so `,` should not be a problem. @clarajeanne please check your import/open setting in libreoffice to only separate by `;` for this file.
Should be removed.